dwave/preprocessing/presolve/cypresolve.pyx

```diff
@@ -52,14 +52,14 @@ cdef class cyPresolver:
         self.cpppresolver.detach_model()
 
     def copy_model(self):
-        """Return a copy of the held CQM."""
+        """Return a copy of the held model."""
```

Comment:
This message as already very terse, but should probably specify in some way what type of model is returned.